src/details/ArborX_DetailsFDBSCANDenseBox.hpp

```diff
@@ -163,13 +163,14 @@ struct FDBSCANDenseBoxCallback
                           CorePointsType const &is_core_point,
                           Primitives const &primitives,
                           DenseCellOffsets const &dense_cell_offsets,
+                          int num_points_in_dense_cells,
```

Comment:
This is not a good change, you introduce a new argument (`num_points_in_dense_cells`) that can be derived from an existing one (`dense_cell_offsets`). Instead, I'd rather pass an execution space in the constructor to do what you want.

Reply:
Just like I did you are suggesting to add another argument.  I don't care either way I can change if you like

Reply:
> Just like I did you are suggesting to add another argument.

Sure, but it's an independent argument, i.e., its value cannot be derived from any other. Please change.

Reply:
Changing.  Note that it would be potentially wasteful if the value is re-used at the call site.